src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <section data-qa="card" class="card">
+
+      <img src="./images/imac.jpeg"
+          alt="The picture of iMac."
+          class="card__img">
+
+      <div class="card__product">
+
+        <p class="card__product-name">APPLE A1419 iMac 27" Retina</p>
```

Comment:
```suggestion
      <div class="card__product">
        <p class="card__product-name">APPLE A1419 iMac 27" Retina</p>
```
no need blank line between parent and child tags
but keep it between sibling tags

fix all similar cases